Src/PerformanceCollector/Shared/QuickPulseTelemetryModule.cs

```diff
@@ -406,6 +428,33 @@ private void CollectionThreadWorker(object state)
                 Thread.Sleep(timeLeftUntilNextTick > TimeSpan.Zero ? timeLeftUntilNextTick : TimeSpan.Zero);
             }
         }
+        
+
+        private void InitializeCollectionThread()
+        {
+            try
+            {
+                this.topCpuCollector.Initialize();
```

Comment:
So if this Initialize method is actually being used to start a thread rather than to merely setup state I would rename it to reflect what it is doing such as "Start", "StartCollecting", "Run", etc.